mobify/platform-scaffold
# Add robots.txt and favicon.ico

Add and serve a robots.txt, since it's possible some Google Search infrastructure wants a valid response for this.

[The original Lancome AMP prototype had this](https://github.com/mobify/amp-proxy/blob/master/public/robots.txt)

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Add `robots.txt`
- Add `favicon.ico`

## How to test-drive this PR
- Run `npm run dev`
- Notice http://localhost:3000/robots.txt returns a valid robots response instead of a 404 page/response.
